chat.c

```diff
@@ -255,6 +256,14 @@ proceed_chat_msg(char type, const char *author, const char *text) {
 		postuser = ">";
 	}
 
+	nicklen = strlen(nick);
+	if (type == 'c')
+		putchar('\a');
+	else if ((s = strstr(text, nick)) != NULL &&
+	    (s == text || !isalpha((unsigned char)s[-1])) &&
+	    !isalpha((unsigned char)s[nicklen]))
```

Comment:
This would trigger for "4op" or "op4" as well as for just "op" or "_op_". I think `isalnum()` would be better here.

Reply:
right!  I've also changed the first check to require a space before the nickname (was notified once by a "no-op" in a message.)  can change back to isalnum() however if you prefer.